Doc/Tutorial/chapter_align.tex

```diff
@@ -928,6 +928,10 @@ \section{Alignment Tools}
 output filename), then invoke this command line via a Python operating system call (e.g.
 using the \texttt{subprocess} module).
 
+\emph{WARNING:} We have decided to drop these command line wrappers in a future Biopython
+release. We will be updating this documentation to instead build the command line
```

Comment:
Don't promise too much..

Reply:
Well, it might turn out to be some examples updated and many examples removed... but I'd hope not.